geode-redis/README.md

```diff
@@ -167,53 +167,53 @@ start server \
-| TYPE                  | SETBIT                                             	| CLUSTER BUMPEPOCH             	|
-| UNSUBSCRIBE           | SETEX                                              	| CLUSTER COUNT-FAILURE-REPORTS 	|
-|                    	| SETNX                                              	| CLUSTER COUNTKEYSINSLOT       	|
-|                    	| SETRANGE                                           	| CLUSTER DELSLOTS     	            |
-|                    	| SHUTDOWN                                           	| CLUSTER FAILOVER              	|
-|                    	| SINTER                                             	| CLUSTER FLUSHSLOTS                |
-|                    	| SINTERSTORE                                        	| CLUSTER FORGET                    |
-|                    	| SISMEMBER                                          	| CLUSTER GETKEYSINSLOT             |
-|                    	| SLOWLOG                                            	| CLUSTER INFO                  	|
-|                    	| SMOVE                                              	| CLUSTER KEYSLOT                   |
-|                    	| SPOP                                               	| CLUSTER MEET                      |
-|                    	| SRANDMEMBER                                        	| CLUSTER MYID                   	|
-|                    	| SSCAN                                              	| CLUSTER NODES                 	|
-|                    	| STRLEN                                             	| CLUSTER REPLICAS                  |
-|                    	| SUNION                                             	| CLUSTER REPLICATE              	|
-|                    	| SUNIONSTORE                                        	| CLUSTER RESET                  	|
-|                    	| TIME                                               	| CLUSTER SAVECONFIG                |
-|                    	| UNLINK [1]                                         	| CLUSTER SET-CONFIG-EPOCH          |
-|                    	|                                       	            | CLUSTER SETSLOT                   |
-|                    	|                                      	                | CLUSTER SLAVES                    |
-|                    	|                                      	                | CLUSTER SLOTS                 	|
-|                    	|                                                       | COMMAND                       	|
-|                    	|                                             	        | COMMAND COUNT                 	|
-|                    	|                                                    	| COMMAND GETKEYS               	|
-|                    	|                                                    	| COMMAND INFO                  	|
+| HSCAN                 | GETRANGE                                              | ACL USERS                     	|    
```

Comment:
We were going to make all doc-related changes as one separate commit, so this can be reverted.

Reply:
ok, rolled back